Bigtable/src/ChunkFormatter.php

```diff
@@ -137,7 +140,7 @@ function ($ex) {
                 }
                 return false;
             },
-            ResumableStream::getMaxRetries($this->options)
+            $this->pluck('retries', $this->options, false) ?: null
```

Comment:
`pluck` already returns null if `retries` isn't found in the array, so the elvis operator is superfluous